gtsam.h

```diff
@@ -2729,8 +2731,6 @@ pair<gtsam::NonlinearFactorGraph*, gtsam::Values*> load2D(string filename,
 pair<gtsam::NonlinearFactorGraph*, gtsam::Values*> load2D(string filename,
     gtsam::noiseModel::Diagonal* model);
 pair<gtsam::NonlinearFactorGraph*, gtsam::Values*> load2D(string filename);
-pair<gtsam::NonlinearFactorGraph*, gtsam::Values*> load2D_robust(string filename,
```

Comment:
Why is this gone?

Reply:
I can add it back, forgot when debugging